docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,91 +1,75 @@
 * git
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
-* access to `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
-* MongoDB (tested with version: {WFM-RC-MongoDBVersion})
-* Redis (tested with version: {WFM-RC-RedisVersion})
+* MongoDB (tested version: {WFM-RC-MongoDBVersion})
+* Redis (tested version: {WFM-RC-RedisVersion})
 
-[id='{context}-published-repositories-cloning-the-git-repositories']
-[discrete]
-== Cloning the Git Repositories
+You must also have access to:
 
-The following commands will clone the `master` branch of each repository and they collectively contain the latest tagged release of the {PROJECT_NAME} project.
+* `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
+* `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
+* `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
 
+[discrete]
+== Installing and Running the {PROJECT_NAME} Applications
+
+1&#46; Clone the Git repositories.
 [source,bash]
 ----
 git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-mobile.git
 git clone git@github.com:feedhenry-raincatcher/raincatcher-portal.git
 ----
 
-Previous versions are available through branches named with the `release-\{semver\}` convention.
-For example, in order to checkout version `0.0.1` of `raincatcher-server`, run the following commands after the cloning:
+Tip: Cloning the repositories provides the latest version. You can use the `git checkout `release-\{semver\}` command if you want a previous version. For example, `git checkout `release-0.0.1`.
 
+2&#46; Download the NPM dependencies for each Git repostory.
 [source,bash]
 ----
-cd raincatcher-server/
-git checkout release-0.0.1
+cd /raincatcher-server
+npm install
 ----
 
-[id='{context}-published-repositories-installing-repository-dependencies']
-[discrete]
-== Installing Repository Dependencies
-
-For each repository previously cloned, run `npm install` so regular NPM dependencies are downloaded.
-
-[id='{context}-published-repositories-starting-mongodb-and-redis']
-[discrete]
-== Starting MongoDB and Redis
-
-. Start MongoDB. For more information, see link:https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/#start-mongod-processes[Start MongoDB Processes].
-
-. Start Redis. For more information, see link:https://redis.io/topics/quickstart#starting-redis[Starting Redis].
+3&#46; Ensure MongoDB and Redis are running.
 
-[id='{context}-published-repositories-starting-the-core-server']
-[discrete]
-== Starting the Core Server
+For more information, see:
+. MongoDB: https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/#start-mongod-processes[Start MongoDB Processes]
+. Redis: https://redis.io/topics/quickstart#starting-redis[Starting Redis]
 
+4&#46; Start the Core server.
```

Comment:
[Question] What's that character? I do not understand the change but looks like -1 for me.

Reply:
it's the html entity for a period. numbered lists in asciidoc are similar to markdown, if you insert a break between two items in the list then numbering starts over. so this is kind of a hack to hard-code numbering for each step in the procedure. it isn't ideal but I haven't found a way around it.

Reply:
it's the html entity for a period. numbered lists in asciidoc are similar to markdown, if you insert a break between two items in the list then numbering starts over. so this is kind of a hack to hard-code numbering for each step in the procedure. it isn't ideal but I haven't found a way around it.

edit: automatically logged in with my other github account on the previous comment. switching to my personal username.